src/circle.tsx

```diff
@@ -53,12 +55,12 @@ export class Circle extends Component<CircleProps, CircleState> {
 
   render() {
     const { text } = this;
-    const { progress, size, bgColor, progressColor, lineWidth, animate, roundedStroke, responsive } = this.props;
+    const { progress, size, bgColor, progressColor, lineWidth, animate, animationDuration, roundedStroke, responsive } = this.props;
     const strokeDashoffset = getOffset(progress);
-    const transition = animate ? 'stroke-dashoffset 1s ease-out' : null;
+    const transition = animate ? 'stroke-dashoffset ' + animationDuration + ' ease-out' : null;
```

Comment:
Should this use ES6 templates instead?

```js
const transition = animate ? `stroke-dashoffset ${animationDuration} ease-out` : null;
```

Reply:
+1 to this, can you please add it as part of this PR?